views/RgbLightView/RgbLightView.tsx

```diff
@@ -42,14 +41,16 @@ export const RgbLightView: FC<RgbLightViewProps> = ({ route }) => {
   return (
     <View style={styles.container}>
       <Navbar text={`${location} - ${nodeId} - ${rgbLight?.name}`} />
-      <View style={styles.content}>
-        <Typography
-          variant="body-medium"
-          text={`Aktualny kolor lampy to: ${rgbLight.values.V_RGB}`}
-        />
-        <Input text={color} onChange={onColorChange} />
-        <Button text="Zmień kolor" onPress={handleChangeColor} />
-      </View>
+      <ScrollView onScrollBeginDrag={Keyboard.dismiss}>
```

Comment:
In my last PR, I moved this ScrollView to a Layout component. You can consider moving onScrollBeginDraw props there

Reply:
Done but not in a prop but just inside scroll view.